packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -139,6 +139,7 @@ export class AzureClient {
             containerSchema,
         );
         const module = { fluidExport: runtimeFactory };
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
         const codeLoader = { load: async () => module };
```

Comment:
You can just fix this if you pull in the `ICodeLoader` type from `@fluidframework/container-definitions` 

```suggestion
        const module: ICodeLoader = { fluidExport: runtimeFactory };
        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
        const codeLoader = { load: async (): ICodeLoader => module };
```